tensorflow/core/platform/default/build_config.bzl

```diff
@@ -152,7 +152,16 @@ def tf_additional_test_deps():
   return []
 
 def tf_additional_test_srcs():
-  return ["platform/default/test_benchmark.cc", "platform/posix/test.cc"]
+  return [
+      "platform/default/test_benchmark.cc",
+  ] + select({
+      "//tensorflow:windows" : [
+          "platform/windows/test.cc"
+        ],
+      "//conditions:default" : [
+          "platform/posix/test.cc",
```

Comment:
windows/test.cc file is introduced in https://github.com/tensorflow/tensorflow/commit/91359470eb2c940733497f2bc6b6e23e3b8d76ef
So I guess it is needed by cmake?